server/api-util/lineItemHelpers.js

```diff
@@ -63,15 +63,13 @@ exports.calculateTotalPriceFromSeats = (unitPrice, unitCount, seats) => {
   if (seats < 0) {
     throw new Error(`Value of seats can't be negative`);
   }
-  const numericPrice = convertMoneyToNumber(unitPrice);
-  const numericTotalPrice = new Decimal(numericPrice)
-    .times(unitCount)
-    .times(seats)
-    .toNumber();
-  return new Money(
-    convertUnitToSubUnit(numericTotalPrice, unitDivisor(unitPrice.currency)),
-    unitPrice.currency
-  );
+
+  const amountFromUnitPrice = getAmountAsDecimalJS(unitPrice);
+  const totalPrice = amountFromUnitPrice.times(unitCount).times(seats);
```

Comment:
I think we need rounding here too as both unitCount (which I assume is the same as units) and seats can be non-integers.